.github/workflows/pytest-workflow.yml

```diff
@@ -117,6 +117,12 @@ jobs:
             tags: controlfreec/makegraph2
           - profile: conda
             tags: coreograph
+          - profile: conda
```

Comment:
Had to add the modules to the conda exclude from pytest. I was getting the error that micromamba is not installed when trying to execute the tests in github actions. Let me know if you have any thoughts about this, or if there is something that I'm missing.